### DISCUSSION TITLE
svcg

### DISCUSSION THREAD

**Discussion by heamx8** [voicepaw/so-vits-svc-fork#690] [Q&A]
svcg : The term 'svcg' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ svcg
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (svcg:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
    
![image](https://github.com/voicepaw/so-vits-svc-fork/assets/111821929/70b8cb7e-0ee9-4794-a1a5-515a76e57b9a)

**Comment by firebait** [comment]
This is for sure a bug but this can launch the GUI:

```python -c "from so_vits_svc_fork.gui import main;main()"```
